src/index.html

```diff
@@ -8,7 +8,205 @@
+                <h2 class="personal-info-cards-title">HOBBIES & INTERESTS</h2>
+                <ul class="personal-info-hobbies-list">
+                  <li class="personal-info-hobbies-item">
+                    <img
+                      class="personal-info-hobbies-item-icon"
+                      src=""
+                      alt="cinema-icon"
+                    />
+                    <span class="personal-info-hobbies-item-label">Cinema</span>
+                  </li>
+                  <li class="personal-info-hobbies-item">
+                    <img
+                      class="personal-info-hobbies-item-icon"
+                      src=""
+                      alt="sport-icon"
+                    />
+                    <span class="personal-info-hobbies-item-label">Sport</span>
+                  </li>
+                </ul>
+              </li>
+
+              <li class="carousel-cards" id="personal-info-eduexp">
+                <h2 class="personal-info-cards-title">Education</h2>
+                <div class="personal-info-eduexp-cards-group">
+                  <div class="personal-info-eduexp-card">
+                    <h3 class="personal-info-eduexp-card-name">
+                      The San Francisco Business School
+                    </h3>
+                    <span class="personal-info-eduexp-card-period"
+                      >2014 - Current</span
+                    >
+                    <span class="personal-info-eduexp-card-desc"
+                      >This is a description of your work experience. Concisely
+                      describe your previous position in a way that’s easy for
+                      readers to scan quickly.</span
+                    >
+                  </div>
+                  <div class="personal-info-edu-card">
+                    <h3 class="personal-info-edu-card-name">
+                      NYC School of Marketing and Advertising
+                    </h3>
+                    <span class="personal-info-edu-card-period"
+                      >2007 - 2020</span
+                    >
+                    <span class="personal-info-edu-card-desc"
+                      >This is a description of your work experience. Concisely
+                      describe your previous position in a way that’s easy for
+                      readers to scan quickly.</span
+                    >
+                  </div>
+                </div>
+              </li>
+
+              <li class="carousel-cards" id="personal-info-eduexp">
+                <h2 class="personal-info-cards-title">EXPERIENCE</h2>
+                <div class="personal-info-eduexp-cards-group">
+                  <div class="personal-info-eduexp-card">
+                    <h3 class="personal-info-eduexp-card-name">
+                      Budget Director
+                      <span class="personal-info-eduexp-card-workplace"
+                        >@Wix.com</span
```

Comment:
If you look the original design carefully, you will find the website is clickable. So the `<a>` should be applied.